client/my-sites/people/followers-list/index.jsx

```diff
@@ -16,290 +16,286 @@ import PeopleListItem from 'calypso/my-sites/people/people-list-item';
 import { Card, Button } from '@automattic/components';
 import classNames from 'classnames';
 import PeopleListSectionHeader from 'calypso/my-sites/people/people-list-section-header';
-import FollowersActions from 'calypso/lib/followers/actions';
-import EmailFollowersActions from 'calypso/lib/email-followers/actions';
 import InfiniteList from 'calypso/components/infinite-list';
-import FollowersData from 'calypso/components/data/followers-data';
-import EmailFollowersData from 'calypso/components/data/email-followers-data';
 import NoResults from 'calypso/my-sites/no-results';
 import EmptyContent from 'calypso/components/empty-content';
-import FollowersStore from 'calypso/lib/followers/store';
-import EmailFollowersStore from 'calypso/lib/email-followers/store';
 import accept from 'calypso/lib/accept';
 import { gaRecordEvent } from 'calypso/lib/analytics/ga';
 import ListEnd from 'calypso/components/list-end';
 import { preventWidows } from 'calypso/lib/formatting';
+import QueryFollowers from 'calypso/components/data/query-followers';
+import {
+	getFollowersForQuery,
+	getIsFetchingFollowersForQuery,
+	getTotalFollowersForQuery,
+} from 'calypso/state/followers/selectors';
+import { removeFollower } from 'calypso/state/followers/thunks';
 
 /**
  * Stylesheet dependencies
  */
 import './style.scss';
+import { connect } from 'react-redux';
```

Comment:
Did we mean to move that up to the `External dependencies` section?

Reply:
done via 5b35750